FileManager/ViewControllers/FilesViewController.swift

```diff
@@ -15,8 +15,14 @@ class FilesViewController: UIViewController {
     
     let segmentControl: UISegmentedControl = UISegmentedControl(items: ["Table", "Collection"])
     
+    var verificationStatus: Bool = false
```

Comment:
There's no need to specify `: Bool`. The type will be infered out of the assigned value (`false`)

Reply:
Fixed.)